squire_lib/src/rounds/mod.rs

```diff
@@ -181,7 +181,7 @@ impl Round {
     /// Calculates the time left in the round, factoring in time extenstions.
     pub fn time_left(&self) -> Duration {
         let length = self.length + self.extension;
-        let elapsed = Duration::from_secs((Utc::now() - self.timer).num_seconds() as u64);
+        let elapsed = Duration::from_secs((Utc::now() - self.timer).num_seconds().max(0) as u64);
```

Comment:
I'm having a hard time finding where this `max` function is coming from, but my intuition tells me that this should be `min`. Can you link the docs for this function?

Reply:
https://doc.rust-lang.org/stable/std/primitive.i64.html#method.max

Gives the max between `self` and `other`. So if `self` is below zero, then the output is zero, I think.

Reply:
Never mind, rust playground is telling me that it's actually the magnitudes that matter, for both `max` and `min`. Might have to use `clamp` instead.

Reply:
Ah, I see. It's a method on `Octal`.